ipynb/chromeos/ChromeOS_Tests.ipynb

```diff
@@ -0,0 +1,827 @@
+    "\n",
+    "```python\n",
+    "import socket\n",
+    "sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)\n",
+    "sock.sendto(\"POWER\", (<TARGET_IP>, 1234))\n",
+    "sock.close()\n",
+    "```\n",
+    "\n",
+    "You should put this right before the instruction that starts the execution of the actual workload in the benchmark."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "reload(logging)\n",
+    "log_fmt = '%(asctime)-9s %(levelname)-8s: %(message)s'\n",
+    "logging.basicConfig(format=log_fmt)\n",
+    "\n",
+    "# Change to info once the notebook runs ok\n",
+    "logging.getLogger().setLevel(logging.DEBUG)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import getpass\n",
+    "from subprocess import Popen, PIPE\n",
+    "\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import scipy.integrate as integrate\n",
+    "import re\n",
+    "import json\n",
+    "\n",
+    "# Support to access the remote target\n",
+    "import devlib\n",
+    "from env import TestEnv\n",
+    "\n",
+    "import trappy\n",
+    "\n",
+    "import socket\n",
+    "\n",
+    "from time import sleep"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Target Configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Define your path to the ChromeOS installation folder\n",
+    "CROS_BASE = \"/data/chromiumos\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Setup a target configuration\n",
+    "my_conf = {\n",
+    "    \n",
+    "    # Target platform and board\n",
+    "    \"platform\"    : 'linux',\n",
+    "    \"board\"       : 'oak',\n",
+    "    \n",
+    "    # Target board IP/MAC address\n",
+    "    \"host\"        : '192.168.0.1',\n",
+    "    \n",
+    "    # Login credentials\n",
+    "    \"username\"    : 'root',\n",
+    "    \"password\"    : 'test0000',\n",
+    "    \n",
+    "    # RTApp calibration values (comment to let LISA do a calibration run)\n",
+    "    \"rtapp-calib\" :  {\n",
+    "        \"0\": 179, \"1\": 177, \"2\": 71, \"3\": 71\n",
+    "    },\n",
+    "    \n",
+    "    # Tools required by the experiments\n",
+    "    #\"tools\"   : [ 'rt-app', 'trace-cmd' ],    \n",
+    "    \"exclude_modules\" : ['hwmon'],\n",
```

Comment:
This should not be required... I do not think we load HWMON by default on OAK... but better to check